### PR TITLE
Per entity manager connection migration config

### PR DESCRIPTION
**Pull request by dave-redfern**
PR for the suggestion: https://github.com/laravel-doctrine/migrations/issues/32

### CODE REVIEW

File: src/Configuration/ConfigurationFactory.php [hunk 1]
```diff
@@ -31,28 +31,34 @@ public function __construct(Repository $config, Container $container)
...
+     * @param string     $name
      *
      * @return Configuration
      */
-    public function make(Connection $connection)
+    public function make(Connection $connection, $name = null)
     {
-        $configuration = new Configuration($connection);
+        if ($name && $this->config->has('migrations.' . $name)) {
+            $config = collect($this->config->get('migrations.' . $name, []));
```
Comment (patrickbrouwers) [inline src/Configuration/ConfigurationFactory.php]: could you change this to `new Collection`

Comment (dave-redfern) [reply src/Configuration/ConfigurationFactory.php]: No problem, and done!

**Comment by patrickbrouwers** [comment]
https://styleci.io/analyses/86wKKM
-> can you fix the CS issues, thanks :)

**Comment by dave-redfern** [comment]
@patrickbrouwers I made the wrong CS fixes. Should be OK now.

**Pull request merged**